keysign/discover.py

```diff
@@ -39,7 +39,7 @@ def start(self):
             key_data = yield threads.deferToThread(self.discovery.find_key, self.userdata)
             log.debug("Received key successfully")
         except ValueError as e:
-            log.debug("Exception occured", exc_info=e)
+            log.debug("foo", exc_info=e)
```

Comment:
I don't think this is a helpful debug message. Do you?  What value do you get when reading "foo" in a log?

Simply add that exc_info parameter to the logging call one line below.

Reply:
Okay, shall I pass it as
`log.debug('Unable to get key_data, received malformed or altered key', exc_info=e)`  
@muelli

Reply:
Yes. In this way, at least, we have a meaningful message before logging the exception.